src/utils/FetchConfig.ts

```diff
@@ -24,112 +24,91 @@ export async function FetchConfig (): Promise<any> {
   sessionStorage.setItem('VUE_ROUTER_BASE', processEnvBaseUrl)
   console.log('Set Vue Router Base to: ' + processEnvBaseUrl)
```

Comment:
While you're in here, could you delete all console.log and console.info statements please? They no longer serve a purpose.

Reply:
On the other hand, I think the `console.info("Initializing ...")` in main.ts are nice to have, so keep those there.

Reply:
Removed the console.log and console.info from this file. I didn't touch others. It seems lots of console logs in other files as well.